session/bootstrap.go

```diff
@@ -970,6 +978,19 @@ func upgradeToVer41(s Session) {
 	doReentrantDDL(s, "ALTER TABLE mysql.user ADD COLUMN `password` TEXT as (`authentication_string`)", infoschema.ErrColumnExists)
 }
 
+// writeDefaultExprPushDownBlacklist writes default expr pushdown blacklist into mysql.expr_pushdown_blacklist
+func writeDefaultExprPushDownBlacklist(s Session) {
+	mustExecute(s, "INSERT HIGH_PRIORITY INTO mysql.expr_pushdown_blacklist VALUES"+
+		"('date_add','tiflash', 'DST(daylight saving time) does not take effect in TiFlash date_add'),"+
+		"('cast','tiflash', 'Behavior of some corner cases(overflow, truncate etc) is different in TiFlash and TiDB')")
+}
+
+func upgradeToVer42(s Session) {
+	doReentrantDDL(s, "ALTER TABLE mysql.expr_pushdown_blacklist ADD COLUMN `store_type` char(100) NOT NULL DEFAULT 'tikv,tiflash,tidb'", infoschema.ErrColumnExists)
+	doReentrantDDL(s, "ALTER TABLE mysql.expr_pushdown_blacklist ADD COLUMN `reason` varchar(200)", infoschema.ErrColumnExists)
```

Comment:
Do we really need this column?

Reply:
I prefer to keep this column, since we will add some default rows in `expr_pushdown_blacklist`, this column can make it self-explained, otherwise, the users may keep asking why those exprs are disabled by default.

Reply:
> I prefer to keep this column, since we will add some default rows in `expr_pushdown_blacklist`, this column can make it self-explained, otherwise, the users may keep asking why those exprs are disabled by default.

Yeah, this explanation is also OK for me.